pkg/build/api/types.go

```diff
@@ -63,6 +63,31 @@ const (
 	// forces the build to be processed by the build controller queue without waiting
 	// for a resync.
 	BuildAcceptedAnnotation = "build.openshift.io/accepted"
+
+	// BuildCreatedEvent is the reason associated with the event registered when a build is created.
+	BuildCreatedEventReason = "BuildCreated"
```

Comment:
They're interesting in an unhealthy system and completely useless / waste space / clog up event views in a normal system.  If the goal is to debug unhealthy, dispatch events for specific unhealthy.  I don't think the serial build thing is a good reason, especially if you have conditions on builds that convey that info, because that's the system working as designed.

We already have an event for created which is the watch.  We don't fire events when anything else is created, so no reason to start here.  We have a long term strategy for user creation stuff which is roughly analogous to an event log for object changes (not just events), and that would subsume this anyway.

In short - I'm really against sending events just because we can.  We should send events that materially improve users understanding of the system.  I'd rather have 2 events that tell people something really important vs 10 that convey the same total amount of info.  Every event is a write to etcd - that cost has to be worth it.